grafeas/client.go

```diff
@@ -114,6 +114,33 @@ func (g *Client) Close() {
 	g.grafeas.Close()
 }
 
+// GetBuildDetails gets BuildDetails for the passed image.
+func (g *Client) GetBuildDetails(ctx context.Context, reference reference.Canonical) (items []voucher.BuildDetail, err error) {
+	filterStr := resourceURL(reference)
```

Comment:
I think this will give us all metadata associated with `reference`, could we change this to 
```suggestion
	filterStr := filterStr = kindFilterStr(reference, common.NoteKind_BUILD)
```

This will only return the BuildDetails associated with `reference`!

Reply:
Done